README.md

```diff
@@ -24,17 +24,27 @@ anon [--config <path to config file, default is ./config.json>]
 Anon is designed to take input from `STDIN` and by default will output the anonymised file to `STDOUT`:
 
 ```sh
-anon < some_file.csv > some_file_anonymised.csv
+anon < some_file > some_file_anonymised
 ```
 
 ### Configuration
 
-In order to be useful, Anon needs to be told what you want to do to each column of the CSV. The config is defined as a JSON file (defaults to a file called `config.json` in the current directory):
+In order to be useful, Anon needs to be told what you want to do to each column/field of the input. The config is defined as a JSON file (defaults to a file called `config.json` in the current directory):
 
 ```json5
 {
   "csv": {
-    "delimiter": ","
+    "delimiter": ",",
+    // Specify in which column a unique ID exists on which the sampling can
+    // be performed. Indices are 0 based, so this would sample on the first
+    // column.
+    "idColumn": "0"
+  },
+  "json": {
```

Comment:
I think it's a bit confusing to have both `csv` and `json` in the same example — should give a bit of preamble and have two examples. Maybe something like this:

> At a high-level, the config looks like this:
> 
> ```json5
> {
>   // Name of the format of the input file — for example, "csv" or "json".
>   "formatName": {
>     // Options for the format you have picked go here.
>     // See the documentation for the format you choose below.
>   },
>   "sampling": {
>     // FIXME: Steal from below.
>   },
>   "actions": [
>     // FIXME: Steal from below.
>   ]
> }
> ```
> 
> ## Formats
>
> You can use CSV or JSON files as input.
> 
> ### CSV
>
> For a CSV file you will need a config like this:
> 
> ```
> "csv": {
>   "delimiter": ",",
>   // Specify in which column a unique ID exists on which the sampling can
>   // be performed. Indices are 0 based, so this would sample on the first
>   // column.
>   "idColumn": "0"
> }
> ```
> 
> ### JSON
>
> For a JSON file you will need to define config like this: 
> ```
> "json": {
>   // Specify in which field a unique ID exists on which the sampling can
>   // be performed.
>   "idField": "id"
> }
> ```